draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -49,32 +50,45 @@ const EmptyState: EmptyState = ({
   children,
   straightCorners,
   useZenStyles,
-}) => (
-  <div
-    className={classnames([
-      styles.container,
-      styles[layoutContext],
-      { [styles.straightCorners]: straightCorners },
-    ])}
-    id={id}
-    data-automation-id={automationId}
-  >
-    <div className={styles.illustrationSide}>
-      <img
-        src={illustrations[illustrationType]}
-        className={styles.illustration}
-      />
-    </div>
+}) => {
+  const theme = useTheme()
+  const inner = (
```

Comment:
What are your thoughts on moving this to a separate variable? I don't really see the need for this abstraction... A good use case for this abstraction (in my mind) is for conditional rendering, complex components with a lot of moving parts, or components that have clear subcomponents.